monkey/infection_monkey/utils/aws_environment_check.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from common.cloud.aws.aws_instance import AwsInstance
+from infection_monkey.telemetry.aws_instance_telem import AwsInstanceTelemetry
+from infection_monkey.utils.threading import create_daemon_thread
+
+logger = logging.getLogger(__name__)
+
+
+def _running_on_aws(aws_instance: AwsInstance) -> bool:
+    return aws_instance.is_instance()
+
+
+def _report_aws_environment():
+    logger.info("Collecting AWS info")
+
+    aws_instance = AwsInstance()
+
+    if _running_on_aws(aws_instance):
+        logger.info("Machine is an AWS instance")
+        AwsInstanceTelemetry({"instance_id": aws_instance.get_instance_id()}).send()
+    else:
+        logger.info("Machine is NOT an AWS instance")
+
+
+def run_aws_environment_check():
```

Comment:
Pass this a `TelemeryMessenger` and send the telemetry via the telemetry messenger instead of using `.send()` on the telemetry instance. It'll be one less thing to fix when we overhaul telemetry.